creation/client/creation.lua

```diff
@@ -59,6 +59,7 @@ AddEventHandler("polyzone:pzcreate", function(zoneType, name, args)
   end
   createdZoneType = zoneType
   drawZone = true
+  disableControlKeyInput()
```

Comment:
One last thing. This is called after drawZone is set to true, but then the function does nothing if drawZone is true. So the controls are not actually being disabled. This call should happen before the previous line. Same for below (line 127) as well.

Reply:
Whoops. Have updated the PR to reflect :)

Reply:
Works for me! Thank you.